clients/venice-push-job/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
     // Exclude transitive dependency
     exclude group: 'org.apache.avro'
     exclude group: 'javax.servlet'
+    exclude group: 'com.fasterxml.jackson'
```

Comment:
From Gaojie's exploration, the offending lib is `com.fasterxml.jackson.core:jackson-databind`. So, do we need to exclude group `com.fasterxml.jackson.core` instead?

Reply:
@huangminchn 
You can run a depdencyInsight to see whether the offending lib is still there or not.

Reply:
@nisargthakkar is right, and we should do:
```
exclude group: 'com.fasterxml.jackson.core'
```

Reply:
Yeah, updated.